tests/pivots/pivot_plugin.test.ts

```diff
@@ -48,4 +48,38 @@ describe("Pivot plugin", () => {
       EMPTY_PIVOT_CELL
     );
   });
+
+  test("getPivotCellFromPosition handles falsy arguments for includeColumnTitle", () => {
+    // prettier-ignore
+    const grid = {
+      A1: "Customer", B1: "Price",
+      A2: "Alice",    B2: "10",    
+      A3: "Bob",      B3: "30",
+    };
+    const model = createModelFromGrid(grid);
+    addPivot(model, "A1:B3", {
+      columns: [{ name: "Customer" }],
+      rows: [{ name: "Price" }],
+      measures: [{ name: "__count", aggregator: "sum" }],
+    });
+    const sheetId = model.getters.getActiveSheetId();
+    setCellContent(model, "C1", "=PIVOT(1,,,false)");
+    expect(model.getters.getPivotCellFromPosition(toCellPosition(sheetId, "D1"))).toMatchObject({
+      measure: "__count",
+      type: "VALUE",
+    });
+    setCellContent(model, "C1", "=PIVOT(1,,,0)");
```

Comment:
Could also test `setCellContent(model, "C1", "=PIVOT(1,,,"FALSE")");` if you want to be complete